bazarr/get_subtitle.py

```diff
@@ -822,6 +831,31 @@ def refine_from_db(path, video):
     return video
 
 
+def refine_from_mediainfo(path, video):
+    if video.fps:
+        return
+
+    exe = get_binary('mediainfo')
+    if not exe:
+        logging.warn('BAZARR MediaInfo library not found!')
```

Comment:
I guess that this will flood the log massively... Maybe we could debug log instead?